modules/gdscript/editor/gdscript_translation_parser_plugin.cpp

```diff
@@ -146,8 +136,7 @@ void GDScriptEditorTranslationParserPlugin::_add_id_ctx_plural(const Vector<Stri
 		return;
 	}
 
-	ids_ctx_plural->push_back(p_id_ctx_plural);
-	ids_ctx_plural_comment.push_back(comment);
+	translations->push_back({ p_id_ctx_plural[0], p_id_ctx_plural[1], p_id_ctx_plural[2], comment });
```

Comment:
Not directly related to this PR, but I would replace `Vector<String>` with a 3-string struct. This code is safe, but relies on the implicit assumption that the vector is always 3-element:

https://github.com/godotengine/godot/blob/e567f49cbb292dd6bb628348e67c1f3c2a3a5f99/modules/gdscript/editor/gdscript_translation_parser_plugin.cpp#L351-L352